lib/buildpacks-ci-pipeline-update-command.rb

```diff
@@ -21,7 +21,7 @@ def run!(concourse_target_name:, pipeline_name:, config_generation_command:, pip
       set-pipeline \
       --pipeline=#{pipeline_prefix}#{pipeline_name} \
       --config=<(#{config_generation_command}) \
-      --load-vars-from=<(lpass show #{buildpacks_configuration.concourse_private_filename} --notes && lpass show #{buildpacks_configuration.deployments_buildpacks_filename} --notes && lpass show #{buildpacks_configuration.repos_private_keys_filename} --notes && lpass show #{buildpacks_configuration.git_repos_private_keys_filename} --notes && lpass show #{buildpacks_configuration.git_repos_private_keys_two_filename} --notes && lpass show #{buildpacks_configuration.bosh_release_private_keys_filename} --notes) \
+      --load-vars-from=private.yml \
```

Comment:
How about `--load-vars-from=<(blackbox_cat private.yml)`? That should hopefully eliminate the temporary file on disk…